src/main/java/com/example/bookstoreapp/servive/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.example.bookstoreapp.servive.impl;
+
+import com.example.bookstoreapp.model.Book;
+import com.example.bookstoreapp.repository.BookRepository;
+import com.example.bookstoreapp.servive.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
```

Comment:
your main class should contain a method which provides a `CommandLineRunner` bean where you test this service methods, check the task description